rpm/droidmedia-devel.spec

```diff
@@ -0,0 +1,32 @@
+Name:          droidmedia-devel
+Summary:       Android media wrapper library development package
+Version:       0.20170214.0
+Release:       1
+Group:         System/Libraries
```

Comment:
Unless I've confused it with something else, I don't think we put a Group in the spec files now.

Reply:
Removed Group from both specs.